<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindowClass</class>
 <widget class="QMainWindow" name="MainWindowClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>期货跟单软件</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QGroupBox" name="MainBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>780</width>
      <height>80</height>
     </rect>
    </property>
    <property name="title">
     <string>主账号</string>
    </property>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>405</x>
       <y>20</y>
       <width>50</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>账号状态</string>
     </property>
     <property name="buddy">
      <cstring>mainStatus</cstring>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>15</x>
       <y>50</y>
       <width>50</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>动态权益</string>
     </property>
     <property name="buddy">
      <cstring>mainBalance</cstring>
     </property>
    </widget>
    <widget class="QLabel" name="label_8">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>50</y>
       <width>50</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>持仓盈亏</string>
     </property>
     <property name="buddy">
      <cstring>mainPositionProfit</cstring>
     </property>
    </widget>
    <widget class="QLabel" name="label_10">
     <property name="geometry">
      <rect>
       <x>405</x>
       <y>50</y>
       <width>50</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>平仓盈亏</string>
     </property>
     <property name="buddy">
      <cstring>mainCloseProfit</cstring>
     </property>
    </widget>
    <widget class="QLineEdit" name="mainStatus">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>460</x>
       <y>20</y>
       <width>110</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>未登录</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="mainBalance">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>50</y>
       <width>110</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>0.00</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="mainPositionProfit">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>265</x>
       <y>50</y>
       <width>110</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>0.00</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="mainCloseProfit">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>460</x>
       <y>50</y>
       <width>110</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>0.00</string>
     </property>
    </widget>
    <widget class="QPushButton" name="beginFollowBtn">
     <property name="geometry">
      <rect>
       <x>600</x>
       <y>20</y>
       <width>70</width>
       <height>50</height>
      </rect>
     </property>
     <property name="text">
      <string>开始跟单</string>
     </property>
    </widget>
    <widget class="QPushButton" name="stopFollowBtn">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>690</x>
       <y>20</y>
       <width>70</width>
       <height>50</height>
      </rect>
     </property>
     <property name="text">
      <string>停止跟单</string>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>25</x>
       <y>20</y>
       <width>40</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>主账号</string>
     </property>
     <property name="buddy">
      <cstring>mainUsername</cstring>
     </property>
    </widget>
    <widget class="QLineEdit" name="mainUsername">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>20</y>
       <width>110</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>20</y>
       <width>50</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>期货公司</string>
     </property>
     <property name="buddy">
      <cstring>mainBrokerName</cstring>
     </property>
    </widget>
    <widget class="QLineEdit" name="mainBrokerName">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>265</x>
       <y>20</y>
       <width>110</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="FollowBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>100</y>
      <width>780</width>
      <height>220</height>
     </rect>
    </property>
    <property name="title">
     <string>样本账号</string>
    </property>
    <widget class="QTableWidget" name="followView">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>760</width>
       <height>190</height>
      </rect>
     </property>
     <property name="editTriggers">
      <set>QAbstractItemView::NoEditTriggers</set>
     </property>
     <property name="selectionBehavior">
      <enum>QAbstractItemView::SelectRows</enum>
     </property>
     <attribute name="horizontalHeaderHighlightSections">
      <bool>false</bool>
     </attribute>
     <attribute name="verticalHeaderVisible">
      <bool>false</bool>
     </attribute>
     <attribute name="verticalHeaderDefaultSectionSize">
      <number>21</number>
     </attribute>
     <column>
      <property name="text">
       <string>序号</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>账号</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>期货公司</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>跟单方向</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>跟单比例</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>账号状态</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>动态权益</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>持仓盈亏</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>平仓盈亏</string>
      </property>
     </column>
    </widget>
   </widget>
   <widget class="QGroupBox" name="ReportBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>330</y>
      <width>780</width>
      <height>240</height>
     </rect>
    </property>
    <property name="title">
     <string>运行日志</string>
    </property>
    <widget class="QTableView" name="eventTable">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>760</width>
       <height>210</height>
      </rect>
     </property>
     <property name="verticalScrollBarPolicy">
      <enum>Qt::ScrollBarAlwaysOn</enum>
     </property>
     <property name="editTriggers">
      <set>QAbstractItemView::NoEditTriggers</set>
     </property>
     <property name="selectionBehavior">
      <enum>QAbstractItemView::SelectRows</enum>
     </property>
     <property name="sortingEnabled">
      <bool>true</bool>
     </property>
     <attribute name="horizontalHeaderHighlightSections">
      <bool>false</bool>
     </attribute>
     <attribute name="verticalHeaderVisible">
      <bool>false</bool>
     </attribute>
     <attribute name="verticalHeaderDefaultSectionSize">
      <number>21</number>
     </attribute>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>账号设置</string>
    </property>
    <addaction name="mainSetting"/>
    <addaction name="followSetting"/>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>跟单设置</string>
    </property>
    <addaction name="strategySetting"/>
   </widget>
   <widget class="QMenu" name="menu_3">
    <property name="title">
     <string>期货公司设置</string>
    </property>
    <addaction name="brokerManage"/>
   </widget>
   <addaction name="menu"/>
   <addaction name="menu_2"/>
   <addaction name="menu_3"/>
  </widget>
  <action name="mainSetting">
   <property name="text">
    <string>主帐号设置</string>
   </property>
  </action>
  <action name="followSetting">
   <property name="text">
    <string>样本账号设置</string>
   </property>
  </action>
  <action name="strategySetting">
   <property name="text">
    <string>跟单策略设置</string>
   </property>
  </action>
  <action name="brokerManage">
   <property name="text">
    <string>服务器管理</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="mainwindow.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>beginFollowBtn</sender>
   <signal>clicked()</signal>
   <receiver>MainWindowClass</receiver>
   <slot>beginFollowBtnClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>679</x>
     <y>102</y>
    </hint>
    <hint type="destinationlabel">
     <x>687</x>
     <y>327</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>stopFollowBtn</sender>
   <signal>clicked()</signal>
   <receiver>MainWindowClass</receiver>
   <slot>stopFollowBtnClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>769</x>
     <y>102</y>
    </hint>
    <hint type="destinationlabel">
     <x>785</x>
     <y>328</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>beginFollowBtnClicked()</slot>
  <slot>stopFollowBtnClicked()</slot>
 </slots>
</ui>
